tests/test_quickstart.py

```diff
@@ -13,20 +13,24 @@
 from sphinx.testing.util import SphinxTestApp
 
 if TYPE_CHECKING:
-    from collections.abc import Callable
     from pathlib import Path
     from typing import Any
 
 warnfile = StringIO()
 
 
-def setup_module():
+def setup_module() -> None:
     disable_colour()
 
 
-def mock_input(
-    answers: dict[str, str], needanswer: bool = False
-) -> Callable[[str], str]:
+class _MockTermInput(Protocol):
```

Comment:
An alternative to much of the work here is to make `term_input` take only keyword arguments:

```python
def term_input(prompt: str, /) -> str:
```

Reply:
I have made this change